app/serializers/question_serializer.rb

```diff
@@ -32,10 +33,6 @@ def required_loan_types
   end
 
   def can_edit
-    if @user
-      Pundit.policy!(@user, object).edit?
-    else
-      return false
-    end
+    object.division_id == selected_division.id
```

Comment:
we should communicate to brendan extra that ppl can only edit qs in the curr division so he can communicate that to tww people really clearly.